doc/can-query-logic.md

```diff
@@ -274,48 +296,62 @@ happen on the data type you pass to your [can-connect can-connect connection]. F
 you might create a `Todo` data type and pass it to a connection like this:
 
 ```js
-import {DefineMap, realtimeRestModel} from "can";
+import {DefineMap, DefineList, realtimeRestModel} from "can";
+import {Todo, todoFixture} from "//unpkg.com/can-demo-models@5";
 
-const Todo = DefineMap.extend({
-  id: {
-    identity: true,
-    type: "number"
-  },
-  complete: "boolean",
-  name: "string"
+// creates a mock todo api
+todoFixture(1);
+
+Todo.list = DefineList.extend("TodoList", {
+  "#": {Type: Todo}
 });
 
-realtimeRestModel({
-  url: "/todos",
+Todo.connection = realtimeRestModel({
+  url: "/api/todos/{id}",
   Map: Todo
 });
+
+Todo.getList().then(todos => {
+  todos.forEach(todo => {
+    console.log(todo.name); // logs todos
+  });
+});
+
 ```
-@highlight 4,15
+@codepen
+@highlight 11-14,only
 
 Internally, `realTimeRest` is using `Todo` to create and configure a `QueryLogic`
 instance for you.  The previous example is equivalent to:
 
 ```js
-import {DefineMap, realtimeRestModel, QueryLogic} from "can";
+import {DefineMap, DefineList, realtimeRestModel, QueryLogic} from "can";
+import {Todo, todoFixture} from "//unpkg.com/can-demo-models@5";
 
-const Todo = DefineMap.extend({
-  id: {
-    identity: true,
-    type: "number"
-  },
-  complete: "boolean",
-  name: "string"
+// creates a mock todo api
+todoFixture(1);
+
+Todo.list = DefineList.extend("TodoList", {
+  "#": {Type: Todo}
 });
 
-var todoQueryLogic = new QueryLogic(Todo);
+const todoQueryLogic = new QueryLogic(Todo);
 
-realtimeRestModel({
-  url: "/todos",
+Todo.connection = realtimeRestModel({
+  url: "/api/todos/{id}",
   Map: Todo,
   queryLogic: todoQueryLogic
 });
+
+Todo.getList().then(todos => {
+  todos.forEach(todo => {
+    console.log(todo.name); // logs todos
+  });
+});
+
 ```
-@highlight 14,19
+@codepen
+@highlight 13-17,only
```

Comment:
This should be highlighting `const todoQueryLogic = new QueryLogic(Todo);` and just the `queryLogic: todoQueryLogic` line.  That is what is important.